packages/dd-trace/src/ritm.js

```diff
@@ -35,6 +35,7 @@ function Hook (modules, options, onrequire) {
   this.modules = modules
   this.options = options
   this.onrequire = onrequire
+  const _origRequire = Module.prototype.require
```

Comment:
Why the underscore prefix?

Reply:
It was to differentiate it from the `origRequire` set at the top of the module. When I used the `origRequire` set at the top of the module for the fix, the unit test does not pass. I think it has something to do with the module being imported and `origRequire` set to `Module.prototype.require` before the test has a chance to patch it with `@azure/functions-core`.

https://github.com/DataDog/dd-trace-js/blob/f736448c1cce901123db52918a141d0dde00d3d4/packages/dd-trace/src/ritm.js#L8

Reply:
If that's the one we consider the original require, shouldn't that be the original require that we always use then, thus replacing the one at the top?

Also, since it's only needed in the code after `if (patchedRequire) return`, it should be moved there.

Reply:
I tried replacing the `origRequire` at the top with this one but it resulted in many other unit tests failing. Adding this one allows the unit tests I added to pass while leaving the other unit tests unaffected.

I'm open to suggestions for how to consolidate these original requires if we can find an approach that is more elegant. This was the best I could find that worked for the new tests and had a minimal impact to the existing code.

Also, I did move `const _origRequire = Module.prototype.require` to be after `if (patchedRequire) return`.